config/cmake/CTestCustom.cmake

```diff
@@ -20,6 +20,7 @@ set (CTEST_CUSTOM_WARNING_EXCEPTION
     ${CTEST_CUSTOM_WARNING_EXCEPTION}
     "note.*expected.*void.*but argument is of type.*volatile"
     "plugin-build.*:[ \t]*warning"
+    "stamp.verify"
```

Comment:
What is this?

Reply:
This file and this section are REGEX for ignoring warnings.